lesson3/hometask.rb

```diff
@@ -0,0 +1,21 @@
+module Myeach
+  def my_each
+    i = 0
+    while i < self.size
+      yield(self[i])
+      i+=1
+    end
+    self
+  end
+end
+
+class Array
+  include Myeach
+end
+
+class Hash
+  include Myeach
+end
+
+[1,2,3].my_each{|i|p i}
```

Comment:
due to [good style practices](https://github.com/bbatsov/ruby-style-guide#spaces-braces):
```ruby
[1,2,3].my_each{|i|p i}
```
should be 
```ruby
[1,2,3].my_each { |i| p i }
```
and
```ruby
[1=>'1', 2=>'2', 3=>'3'].my_each{|i|p i}
```
should be 
```ruby
{1 => '1', 2 => '2', 3 => '3'}.my_each { |i| p i }
```